modules/identity-provider-about-request-header.adoc

```diff
@@ -7,7 +7,9 @@
 
 A request header identity provider identifies users from request
 header values, such as `X-Remote-User`. It is typically used in combination with
-an authenticating proxy, which sets the request header value.
+an authenticating proxy, which sets the request header value. By its nature, the
```

Comment:
"By its nature," isn't necessary, can you remove that part? 

Otherwise this looks good to me!

Reply:
Updated. Thanks for spotting this!